apps/files/src/views/ReferenceFileWidget.vue

```diff
@@ -21,50 +21,155 @@
 
 <template>
 	<div v-if="!accessible" class="widget-file widget-file--no-access">
-		<div class="widget-file--image widget-file--image--icon icon-folder" />
-		<div class="widget-file--details">
-			<p class="widget-file--title">
+		<span class="widget-file__image widget-file__image--icon">
+			<FolderIcon v-if="isFolder" :size="88" />
+			<FileIcon v-else :size="88" />
+		</span>
+		<span class="widget-file__details">
+			<p class="widget-file__title">
 				{{ t('files', 'File cannot be accessed') }}
 			</p>
-			<p class="widget-file--description">
+			<p class="widget-file__description">
 				{{ t('files', 'The file could not be found or you do not have permissions to view it. Ask the sender to share it.') }}
 			</p>
-		</div>
+		</span>
 	</div>
+
+	<!-- Live preview if a handler is available -->
+	<component :is="viewerHandler.component"
+		v-else-if="viewerHandler && !failedViewer"
```

Comment:
```suggestion
		v-else-if="interactive && viewerHandler && !failedViewer"
```

To make sure we load the fallback if interactive view was not requested

Reply:
On it! :+1: